config/prod/nf-syn23664726-s3.yaml

```diff
@@ -0,0 +1,29 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/s3-bucket.yaml"
```

Comment:
This seems old, should we not use [s3-external-bucket.j2 ?](https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/S3/synapse-external-bucket.j2)

Reply:
Yep, I was using an old template it seems!